src/omnicore/notifications.h

```diff
@@ -3,26 +3,48 @@
 
 #include <stdint.h>
 #include <string>
+#include <vector>
+
+//! Alert types
+#define ALERT_FEATURE_UNSUPPORTED     0 // not allowed via alert broadcast
```

Comment:
Yeah, you can probably make a case for both sides (or `enum`). The upside of `const T` is that it's contained by namespaces, which can be desired. For example, we could have something like:
```c++
namespace omnicore {
namespace rules {
const unsigned int max_multisig_payloads = 2;
const unsigned int max_sequence_number = 255;
}
}
```
I have no strong opinion about this though. :)